rosdep/python.yaml

```diff
@@ -1057,6 +1057,11 @@ python-oauth2:
   debian: [python-oauth2]
   fedora: [python-oauth2]
   ubuntu: [python-oauth2]
+python-oauth2client:
+  debian: [python-oauth2client]
+  fedora: [python-oauth2client]
+  ubuntu:
+    trusty: [python-oauth2client]
```

Comment:
This page won't show packages in EOL Ubuntu distro which we should still list here if the package is available for them.